.gitignore

```diff
@@ -5,7 +5,7 @@
 *.so
 *.dylib
 /shadomplexer-go
-/mmp-go
+/mmp-go*
```

Comment:
you may also want to add *.exe

Reply:
https://github.com/Qv2ray/mmp-go/blob/2e803babf22f1824e7f0d6a71a3010bad28d1b5b/.gitignore#L2